sites/public/layouts/eligibility.tsx

```diff
@@ -19,7 +59,7 @@ const EligibilityLayout = (props) => {
           <div className="bg-gray-200">
             <div className={`mx-auto ${styles.eligibility_contents}`}>
               <ProgressNav
-                currentPageSection={props.currentPageSection}
+                currentPageSection={props.currentPage + 1}
```

Comment:
Why is this +1 now?

Reply:
For whatever reason, `ProgressNav` is 1 indexed, whereas I (correctly and sanely) decided to 0 index the eligibility pages.